.circleci/config.yml

```diff
@@ -17,14 +17,15 @@ executors:
     docker:
       - image: circleci/openjdk:11-jdk-browsers
       - image: localstack/localstack:0.10.5
+        environment:
+          SERVICES: sqs
+          DATA_DIR: /tmp/localstack/data
+          DOCKER_HOST: unix:///var/run/docker.sock
+          AWS_EXECUTION_ENV: True
+          DEFAULT_REGION: eu-west-2
+          TMPDIR: /private
     environment:
-      _JAVA_OPTIONS: -Xmx192m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -XX:ParallelGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false
-      SERVICES: sqs
-      DATA_DIR: /tmp/localstack/data
-      DOCKER_HOST: unix:///var/run/docker.sock
-      AWS_EXECUTION_ENV: True
-      DEFAULT_REGION: eu-west-2
-      TMPDIR: /private
+      _JAVA_OPTIONS: -Xmx256m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -XX:ParallelGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false -Dorg.gradle.jvmargs=-XX:+UseContainerSupport
```

Comment:
The most important here seems to be ` -Dorg.gradle.jvmargs=-XX:+UseContainerSupport`, without which my builds kept on failing

Reply:
Interesting!